src/components/Dialogs/ActionDialog/ActionDialog.style.ts

```diff
@@ -1,15 +1,25 @@
 import styled from '@emotion/styled'
-import { breakpoints, sizes } from '@/shared/theme'
+import { css } from '@emotion/react'
+import { breakpoints, sizes, colors, typography } from '@/shared/theme'
+import { Button } from '@/shared/components'
 
 const BREAKPOINT = breakpoints.small
 
+type ButtonProps = {
+  error?: boolean
+  warning?: boolean
+}
+
 export const ButtonsContainer = styled.div`
   display: flex;
   flex-direction: column;
 
   > * + * {
     margin-top: ${sizes(2)};
   }
+  button {
+    font-weight: ${typography.weights.bold};
```

Comment:
Looking at Figma all buttons should have 700 weight, so if that's not what's in the component currently, we should update it